tests/common/snappi_tests/common_helpers.py

```diff
@@ -990,8 +990,8 @@ def get_egress_queue_count(duthost, port, priority):
     # If DUT is multi-asic, asic will be used.
     if duthost.is_multi_asic:
         asic = duthost.get_port_asic_instance(port).get_asic_namespace()
-        raw_out = duthost.shell("sudo ip netns exec {} show queue counters {} | sed -n '/UC{}/p'".
-                                format(asic, port, priority))['stdout']
+        raw_out = duthost.shell("sudo show queue counters {} -n {} | sed -n '/UC{}/p'".
```

Comment:
The CLI should work fine without the 'sudo' as well.

Reply:
Thanks, will adjust the script